doc/modules/cross_validation.rst

```diff
@@ -261,25 +261,25 @@ two slightly unbalanced classes::
   [0 1 2 4 5 6 7] [3 8 9]
 
 
-Label k-fold
```

Comment:
Indeed. This way we can have a general explanation of what we mean by "grouped data" with a nice example.  I'd prefer it to be example-driven. Currently the example in the group k-fold description is a bit of an afterthought. 

Instead we can lead with the grouped subjects example, and explain why, for a fair evaluation of generalization, we must assume that test time *subjects* are never seen before, not just individual samples. Then explain the classes available to accomplish this.